conceptnet5/readers/cc_cedict.py

```diff
@@ -204,38 +195,37 @@ def handle_file(filename, output_file):
                     out.write(edge)
                 continue
 
-            # Handle abbreviations
-            if re.match(ABBR_REGEX, definition):
-                abbreviations = extract_abbreviations(definition)
-                if abbreviations:
-                    for abbr in abbreviations:
-                        edge = make_edge(rel='/r/RelatedTo',
-                                         start=standardized_concept_uri('zh-Hant', traditional),
-                                         end=standardized_concept_uri('zh', abbr),
-                                         dataset=DATASET,
-                                         license=LICENSE,
-                                         sources=SOURCE)
-                        out.write(edge)
-
-                        edge = make_edge(rel='/r/RelatedTo',
-                                         start=standardized_concept_uri('zh-Hans', simplified),
-                                         end=standardized_concept_uri('zh', abbr),
-                                         dataset=DATASET,
-                                         license=LICENSE,
-                                         sources=SOURCE)
-                        out.write(edge)
-                continue
+            if re.match(SEE_ALSO_REGEX, definition):
+                references = extract_han_characters(definition)
+                for reference in references:
+                    edge = make_edge(rel='/r/RelatedTo',
+                                     start=standardized_concept_uri('zh-Hant', traditional),
+                                     end=standardized_concept_uri('zh', reference),
+                                     dataset=DATASET,
+                                     license=LICENSE,
+                                     sources=SOURCE)
+                    out.write(edge)
+
+                    edge = make_edge(rel='/r/RelatedTo',
+                                     start=standardized_concept_uri('zh-Hans', simplified),
+                                     end=standardized_concept_uri('zh', reference),
+                                     dataset=DATASET,
+                                     license=LICENSE,
+                                     sources=SOURCE)
+                    out.write(edge)
 
             # Remove 'lit.', 'fig.'
             definition = LIT_FIG_REGEX.sub('', definition)
 
             # Expand sth and sb
-            definition = definition.replace('sth', 'something')
-            definition = definition.replace('sb', 'someone')
+            definition = definition.replace(' sth ', 'something')
```

Comment:
I think this would fail for two reasons. It doesn't match words at the end of the definition, and it deletes the spaces surrounding the word.

This would convert the definition "to give sth to sb" to "to givesomethingto sb".